posthog/urls.py

```diff
@@ -92,6 +93,32 @@ def setup_admin(request):
         })
         return redirect('/')
 
+def social_create_user(strategy, details, backend, user=None, *args, **kwargs):
+    if user:
+        return {'is_new': False}
+
+    signup_token = strategy.session_get('signup_token')
+    if signup_token is None:
```

Comment:
social auth will shortcircuit when a user is attempting to create an account without a team token. So, not socially authed users will exist without a team association

Reply:
In terms of the error message here, I'm guessing you can get here if you're just trying to log in but don't actually have an account? In that case the error message is a bit confusing.

I think a user wouldn't come here if they signed up via the team signup page, so the user message could be something like "There's no account associated with that account" or something like that.